libr/anal/fcn.c

```diff
@@ -851,7 +851,7 @@ static int fcn_recurse(RAnal *anal, RAnalFunction *fcn, ut64 addr, ut64 len, int
 				r_anal_block_set_size (bb, (ut64)addrbytes * (ut64)delay.after);
 				fcn->ninstr--;
 				if (anal->verbose) {
-					eprintf ("Correct for branch delay @ %08"PFMT64x " bb.addr=%08"PFMT64x " corrected.bb=%"PFMT64u" f.uncorr=%"PFMT64u"\n",
+					R_LOG_ERROR ("Correct for branch delay @ %08"PFMT64x " bb.addr=%08"PFMT64x " corrected.bb=%"PFMT64u" f.uncorr=%"PFMT64u,
```

Comment:
same here

Reply:
<img width="184" alt="image" src="https://user-images.githubusercontent.com/3628862/199478175-f41ba573-601e-4f5d-89c2-75627e8b8a60.png">
so anal->verbose should be removed and here as well?